content/en/_index.html

```diff
@@ -1,32 +1,32 @@
 ---
-title: Cozystack - Free PaaS platform and framework for building clouds
+title: Cozystack - Free platform-as-a-service and framework for building clouds
```

Comment:
We don't offer platform-as-a-service, instead Cozystack is positioned as platform for providing managed services.

Maybe it would be better to say something like this:
- `Cozystack - Free platform and framework for building clouds`
- `Cozystack - Free platform and framework for managed services` and so on.
- `Cozystack - Free PaaS framework for building clouds`

Reply:
resolved